packages/iTwinUI-react/src/core/Stepper/Stepper.tsx

```diff
@@ -2,78 +2,63 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
-import cx from 'classnames';
 import React from 'react';
-import { useTheme } from '../utils';
+import { Step, StepCustomProps, useTheme } from '../utils';
 import '@itwin/itwinui-css/css/stepper.css';
-import { Step } from './Step';
 
-export type WizardLocalization = {
+export type StepperLocalization = {
   stepsCountLabel: (currentStep: number, totalSteps: number) => string;
 };
 
-export type WizardType = 'default' | 'long' | 'workflow';
-
-export type StepProperties = {
-  /**
-   * The title/label of the step.
-   */
-  name: string;
-  /**
-   * A tooltip giving detailed description to this step.
-   */
-  description?: string;
-};
-
-export type WizardProps = {
+export type StepperProps = {
   /**
    * Current step index, 0 - based.
    */
   currentStep?: number;
   /**
    * An array of step objects.
    */
-  steps: StepProperties[];
+  steps: StepCustomProps[];
   /**
-   *  The type of Wizard to display.
+   *  The type of Stepper to display.
    *  @default 'default'
    */
-  type?: WizardType;
+  type?: 'default' | 'long';
   /**
    *  Option to provide localized strings.
    */
-  localization?: WizardLocalization;
+  localization?: StepperLocalization;
   /**
    *  Click handler on completed step.
    */
   onStepClick?: (clickedIndex: number) => void;
 };
 
-const defaultWizardLocalization: WizardLocalization = {
+const defaultStepperLocalization: StepperLocalization = {
   stepsCountLabel: (currentStep, totalSteps) =>
     `Step ${currentStep} of ${totalSteps}:`,
 };
 
 /**
- * A wizard displays progress through a sequence of logical and numbered steps.
+ * A stepper displays progress through a sequence of logical and numbered steps.
  * It may also be used for navigation.
  *
  * The `type` can be set to 'long' to show labels under steps.
  *
  * @example
- * <Wizard
+ * <Stepper
  *  steps=[{name: "Step One"}, {name: "Step Two"}, {name: "Step Three"}]
  *  currentStep={0}
  *  type='long'
  *  />
  */
-export const Wizard = React.forwardRef<HTMLDivElement, WizardProps>(
```

Comment:
We cannot just remove the existing component or its props as that would constitute a breaking change. So we definitely need to keep the "old" component until we hit 2.0. 

For this PR, I would suggest only changing the classnames inside `Wizard` and making no other changes. Or at the very least don't export `Stepper` and `WorkflowDiagram` publicly until we discuss the public API. Would be much easier to review that way.